apps/platforms/win/wsl/ubuntu.talon

```diff
@@ -1,11 +1,12 @@
 app: ubuntu
```

Comment:
I believe this app definition got remove?

Reply:
Since this is no longer ubuntu-specific, it may make sense to rename to wsl.talon

Reply:
Right. I removed that line but I'm not sure I'm not sure about moving the contents of this file into `wsl.talon`. Because I'm not sure if the definitions here are appropriate to all possible wsl distros, though they certainly are applicable to more than just Ubuntu.

There are configurations one can run under wsl which would not include the `/mnt` auto-mounts or all the commands controlled by the `user.file_manager`, `user.generic_unix_shell` or `user.git` tags. In fact, I think it might be the more usual case for many docker-desktop instances. Also, I wonder if future distros will all necessarily conform in this regard.

I suppose we could redefine the `go <user.letter>` using an action, and that way it could always be redefined for less generic contexts. I guess we could also use actions to set the tags. Is that the right way to go?

We could follow the 'generic terminal' pattern of a multi-level hierarchy. That is, we could rename `ubuntu.talon` to something like `generic_linix.talon` or `generic_wsl.talon`. Then, for non-conforming wsl contexts, new .talon files could always be added to override the generic definitions.

Just thinking out loud...